diff --git a/src/Configurations/10-main.conf b/src/Configurations/10-main.conf
index 8dc3e858dfa..8e8ae4537ef 100644
--- a/src/Configurations/10-main.conf
+++ b/src/Configurations/10-main.conf
@@ -1309,7 +1309,8 @@ my %targets = (
         bn_asm_src       => sub { return undef unless @_;
                                   my $r=join(" ",@_); $r=~s|asm/x86_64-gcc|bn_asm|; $r; },
         perlasm_scheme   => "auto",
-        multilib         => "-x64",
+        multilib         => picker(release => "",
+                                   debug => "d"),
     },
     "VC-WIN32" => {
         inherit_from     => [ "VC-noCE-common", asm("x86_asm"),
@@ -1324,6 +1325,8 @@ my %targets = (
         # "WOW" stands for "Windows on Windows", and "VC-WOW" engages
         # some installation path heuristics in windows-makefile.tmpl...
         build_scheme     => add("VC-WOW", { separator => undef }),
+        multilib         => picker(release => "",
+                                   debug => "d"),
     },
     "VC-CE" => {
         inherit_from     => [ "VC-common" ],
