diff --git a/src/Configurations/10-main.conf b/src/Configurations/10-main.conf
index 468698a3..c5fc9cff 100644
--- a/src/Configurations/10-main.conf
+++ b/src/Configurations/10-main.conf
@@ -1420,7 +1420,8 @@ my %targets = (
         uplink_arch      => 'x86_64',
         asm_arch         => 'x86_64',
         perlasm_scheme   => sub { vc_win64a_info()->{perlasm_scheme} },
-        multilib         => "-x64",
+        multilib         => picker(release => "",
+                                   debug => "d"),
     },
     "VC-WIN32" => {
         inherit_from     => [ "VC-noCE-common" ],
@@ -1436,6 +1437,8 @@ my %targets = (
         # "WOW" stands for "Windows on Windows", and "VC-WOW" engages
         # some installation path heuristics in windows-makefile.tmpl...
         build_scheme     => add("VC-WOW", { separator => undef }),
+        multilib         => picker(release => "",
+                                   debug => "d"),
     },
     "VC-CE" => {
         inherit_from     => [ "VC-common" ],
